spec/controllers/hearings/hearing_day/filled_hearing_slots_controller_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+RSpec.describe Hearings::HearingDay::FilledHearingSlotsController, :all_dbs, type: :controller do
```

Comment:
```suggestion
RSpec.describe Hearings::HearingDay::FilledHearingSlotsController, type: :controller do
```
I think Ferris told me these aren't necessary any more?

Reply:
added the change